src/bodywork/k8s/secrets.py

```diff
@@ -193,12 +212,23 @@ def delete_secret(namespace: str, name: str) -> None:
     k8s.CoreV1Api().delete_namespaced_secret(namespace=namespace, name=name)
 
 
+def delete_secret_group(namespace: str, group: str) -> None:
+    """Delete a group of secrets in a namespace.
+
+    :param namespace: Namespace in which to look for the secret to
+    delete.
```

Comment:
Should be indented.

Reply:
Should all be on the same line, fixed this and the comment for the delete secrets method as well.